frontend/src/commons/Buttons.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 import styled from "styled-components/macro";
-import Button from "@material-ui/core/Button";
-import ButtonGroup from '@material-ui/core/ButtonGroup';
 
-export default function Buttons({labelButtonOne, labelButtonTwo, labelButtonThree,
-                                    disableButtonOne = true, disableButtonTwo = true, disableButtonThree = true,
-                                clickHandlerOne, clickHandlerTwo, clickHandlerThree}) {
+export default function Buttons({labelButtonOne, labelButtonTwo, labelButtonThree, lableButtonFour,
+                                    disableButtonOne = true, disableButtonTwo = true, disableButtonThree = true, disableButtonFour = true,
+                                clickHandlerOne, clickHandlerTwo, clickHandlerThree, clickHandlerFour}) {
     return (
-        <ButtonGroupStyled variant="text" color="primary" aria-label="text primary button group">
-            <Button disabled={disableButtonOne} onClick={clickHandlerOne}>{labelButtonOne}</Button>
-            <Button disabled={disableButtonTwo} onClick={clickHandlerTwo}>{labelButtonTwo}</Button>
-            <Button disabled={disableButtonThree} onClick={clickHandlerThree}>{labelButtonThree}</Button>
+        <ButtonGroupStyled>
+            <ButtonStyled disabled={disableButtonOne} onClick={clickHandlerOne}>{labelButtonOne}</ButtonStyled>
+            <ButtonStyled disabled={disableButtonTwo} onClick={clickHandlerTwo}>{labelButtonTwo}</ButtonStyled>
+            <ButtonStyled disabled={disableButtonThree} onClick={clickHandlerThree}>{labelButtonThree}</ButtonStyled>
+            <ButtonStyled disabled={disableButtonFour} onClick={clickHandlerFour}>{lableButtonFour}</ButtonStyled>
```

Comment:
Can you find a better name for this component? Just from reading the code I have no clue what they are used for. Do you really need a totally generic group of 4 buttons? Imho having one button component and rendering it where and when it is needed is preferable.

Reply:
Good point! :)